shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -467,6 +469,9 @@
 
           // Insert placeholders to fill remaining space
           var remaining = this.layout.cols - x;
+          if (isRTL) {
+            x = (windowWidth - this.layout.gridItemWidth) - x;
```

Comment:
I hadn't thought of this before, but this assumes that the grid will take the whole window (which it will in all of our use-cases, but still) - this could be expressed as (this.layout.gridItemWidth * (this.layout.cols - 1)) - x; Alternatively, change windowWidth to gridWidth and calculate that above instead of getting the window width (there may already be a layout function for this, I don't recall off-hand).